test/qunit/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <script src="../../dist/js-reporters.js"></script>
+    <link href="../../node_modules/qunitjs/qunit/qunit.css" rel="stylesheet" type="text/css"/>
```

Comment:
Notice this a little late: Remove `type` attribute, remove closing slash (unnecessary on empty element), see also http://contribute.jquery.org/style-guide/html/

Should also start with `<link>` before any `<script>`.